src/test/java/teammates/test/cases/common/InstructorPrivilegesTest.java

```diff
@@ -441,7 +439,46 @@ public void testValidatePrivileges() {
         assertTrue(privileges.isAllowedForPrivilege(sectionName, Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS));
         assertTrue(privileges.isAllowedForPrivilege(sectionName, sessionName, Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS));
     }
-    
+
+    @Test
+    public void testhasDefaultPrivileges() {
```

Comment:
`testHas...` or you can follow the new test naming convention `hasDefaultPrivileges`